integration_tests/tests/integration_tests/snapshots/integration_tests__test_show_collection_metadata_option__get_asset_with_show_collection_metadata_option.snap

```diff
@@ -0,0 +1,75 @@
+---
+source: integration_tests/tests/integration_tests/test_show_collection_metadata_option.rs
+expression: response
+snapshot_kind: text
+---
+{
+  "interface": "ProgrammableNFT",
+  "id": "AH6wj7T8Ke5nbukjtcobjjs1CDWUcQxndtnLkKAdrSrM",
+  "content": {
+    "$schema": "https://schema.metaplex.com/nft1.0.json",
+    "json_uri": "https://madlads.s3.us-west-2.amazonaws.com/json/1983.json",
+    "files": [],
+    "metadata": {
+      "name": "Mad Lads #1983",
+      "symbol": "MAD",
+      "token_standard": "ProgrammableNonFungible"
+    },
+    "links": {}
+  },
+  "authorities": [
+    {
+      "address": "2RtGg6fsFiiF1EQzHqbd66AhW7R5bWeQGpTbv2UMkCdW",
+      "scopes": [
+        "full"
+      ]
+    }
+  ],
+  "compression": {
+    "eligible": false,
+    "compressed": false,
+    "data_hash": "",
+    "creator_hash": "",
+    "asset_hash": "",
+    "tree": "",
+    "seq": 0,
+    "leaf_id": 0
+  },
+  "grouping": [
+    {
+      "group_key": "collection",
+      "group_value": "J1S9H3QjnRtBbbuD4HjPV6RpRhwuk4zKbxsnCHuTgh9w",
+      "collection_metadata": {}
```

Comment:
Shouldn't there be collection metadata here?

Reply:
It should but the metadata json uri isn't being fetched. Is there a helper in specs for fetching this? Can spoof it by writing info to the db so can be looked up by the API call.

Reply:
ah yeah you're right even base metadata not downloaded here.  its fine for integ test marking as resolved.